pkg/transfers/inbound/cleanup_test.go

```diff
@@ -43,3 +72,149 @@ func TestCleanupErr(t *testing.T) {
 		t.Errorf("unexpected deleted file: %s", agent.DeletedFile)
 	}
 }
+
+func Test_CleanupEmptyFiles_InboundPath_Success(t *testing.T) {
+	agent := &upload.MockAgent{}
+
+	dir, _ := ioutil.TempDir("", "clenaup-testing")
```

Comment:
All instances of `clenaup-testing` should be `cleanup-testing`

Reply:
Totally. Good catch.